app/src/cambodia/res/values/donottranslate.xml

```diff
@@ -32,9 +32,35 @@
     <string name="confirmInvalidQuestionUID" translatable="false"></string>
     <string name="residenceVillageUID" translatable="false"></string>
     <string name="patientResidenceVillageOtherCode" translatable="false"></string>
-
     <string name="control_data_element_phone_metadata" translatable="false">RuNZUhiAmlv</string>
     <string name="control_data_element_datetime_capture" translatable="false">qWMb2UM2ikL</string>
     <string name="control_data_element_datetime_sent" translatable="false">aBahytzj2u0</string>
-    <string name="category_cc" translatable="false"></string>
+    <string name="error_no_treatment" translatable="false"> </string>
+
+    <string name="nonexistent" translatable="false">nonexistent</string>
+    <string name="dynamicTreatmentHideQuestionUID" translatable="false">@string/nonexistent</string>
```

Comment:
@idelcano if there is no treatment matrix for lao and cambodia, do not you think it would be better not to contaminate them with treatment matrix strings and code? I think it would be better if all that has to do with treatment matrix exists within strategy only for myanmar.